### DISCUSSION TITLE
Issues with Reface DX

### DISCUSSION THREAD

**Discussion by cmdrcryptid** [christofmuc/KnobKraft-orm#332] [Q&A]
So recently, I started trying to manage my reface DX presets with knobcraft orm, and every time I click any patch in the librarian, I get the following messages in the terminal window.

"20:52:40: info Sending program change to Yamaha Reface DX for patch Init Voice: program Bank1-2 [known bank].
20:52:40: error Failed to build MIDI bank and program change messages for Yamaha Reface DX, program error?"

I have the reface connected to the computer via the USB out from the back of the synth itself, & it recognizes the synth is there, & it recognizes all the names of the patches, it just can't read or write any information to the patches. What's going on here, and what can I do to fix it?

**Comment by christofmuc** [comment]
Hm, are you using the newest version? There you can try to select the send mode (a combo box below the grid) and set it to "sysex". It should then try to send the patch instead of a program change.

I need to see why it is complaining about the bank change. Thanks for reporting!

**Reply by christofmuc** [reply]
Ok, confirmed as a bug!

There is a workaround: Set the "send mode" to "edit buffer" (the box is just below the patch grid in the lower left corner), then it should work. I'll try to fix the bug with the next release, thanks for reporting!

**Reply by christofmuc** [reply]
I released 2.4.0 which should fix this problem!

**Comment by christofmuc** [comment]
Closing this as fixed
